### ISSUE TITLE
Login Whatsapp api

### ISSUE THREAD

**Issue by amec1234** [WHAnonymous/Chat-API#1409]
When I'm trying to login. I always get the same error. Even though the number is not blocked. 

`Fatal error: Uncaught exception 'LoginFailureException' in /Applications/..... Stack trace: #0 /Applications/..../whatsprot.class.php(551): WhatsProt->doLogin() #1 /Applications/....exampleFunctional.php(67): WhatsProt->loginWithPassword('xxxx...') #2 {main} thrown in /Applications/...whatsprot.class.php on line 2270`

**Comment by mgp25** [comment]
@amec1234 are you using latest API code? If you are using latest code, then make sure if the number is not blocked (you can use `blockChecker.php`). If its not blocked, then its your IP

Regards

**Comment by amec1234** [comment]
is there any chance that the ip gets unblocked? Like waiting some days until trying again?

**Comment by 0xTryCatch** [comment]
Nobody knows

**Comment by borjapu** [comment]
Hello , i'm having the same problem, 
with the new API i can't send any message. error:'Uncaught exception 'LoginFailureException''
But with the old API i can. 
The problem was , that with the old API i cant recive and with newest i can. 
I dont understand why i can send with old api and no with the last. 
Some help?, 
thanks in advance

**Comment by mgp25** [comment]
@borjapu open a new issue, post **full** debug log and code
